Core/include/Acts/Surfaces/AnnulusBounds.hpp

```diff
@@ -154,7 +154,6 @@ class AnnulusBounds : public DiscBounds {
   Vector2D m_moduleOrigin;
   Vector2D m_shiftXY;  // == -m_moduleOrigin
   Vector2D m_shiftPC;
-  double m_phiAvg;
```

Comment:
This was the bug mentioned, we used undefined `m_phiAvg` which resulted in useless results.